.github/workflows/build-publish.yaml

```diff
@@ -72,43 +56,41 @@ jobs:
           docker build --file Dockerfile -t $GITHUB_REPOSITORY:$IMAGE_TAG .
 
       - name: Save image as artifact
-        env:
-          IMAGE_TAG: ${{ needs.prepare.outputs.FULL_IMAGE_TAG }}
         run: |
-          docker save -o /tmp/container.tar.gz $GITHUB_REPOSITORY:$IMAGE_TAG
+          docker save -o /tmp/container-$IMAGE_TAG.tar.gz $GITHUB_REPOSITORY:$IMAGE_TAG
```

Comment:
This might actually save a lot of container images as authors add commits to a PR. Should we maybe use `GITHUB_REF_NAME` name instead?